### PR TITLE
[i18n] Added missing UI translations

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | N/A
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/translations/ezrepoforms_content_type.en.xliff [hunk 1]
```diff
@@ -7,33 +7,33 @@
...
-        <source>field_definition.ezobjectrelation.selection_root_udw_button</source>
-        <target state="new">field_definition.ezobjectrelation.selection_root_udw_button</target>
+        <source>Select item</source>
+        <target state="new">Select item</target>
         <note>key: field_definition.ezobjectrelation.selection_root_udw_button</note>
       </trans-unit>
       <trans-unit id="d892163a41d816c03db3c4a04e35550f02169c11" resname="field_definition.ezobjectrelation.selection_root_udw_title">
-        <source>field_definition.ezobjectrelation.selection_root_udw_title</source>
-        <target state="new">field_definition.ezobjectrelation.selection_root_udw_title</target>
+        <source>Select a start location for browsing for relation</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/ezrepoforms_content_type.en.xliff]: `Select a starting location`

File: src/bundle/Resources/translations/ezrepoforms_content_type.en.xliff [hunk 2]
```diff
@@ -7,33 +7,33 @@
...
-        <source>field_definition.ezobjectrelationlist.selection_root_udw_button</source>
-        <target state="new">field_definition.ezobjectrelationlist.selection_root_udw_button</target>
+        <source>Select location</source>
+        <target state="new">Select location</target>
         <note>key: field_definition.ezobjectrelationlist.selection_root_udw_button</note>
       </trans-unit>
       <trans-unit id="17d8f3e8bbd5b5f0f5b4b02ddaeaf2af82b0d222" resname="field_definition.ezobjectrelationlist.selection_root_udw_title">
-        <source>field_definition.ezobjectrelationlist.selection_root_udw_title</source>
-        <target state="new">field_definition.ezobjectrelationlist.selection_root_udw_title</target>
+        <source>Select a start location for browsing for relations</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/ezrepoforms_content_type.en.xliff]: `Select a starting location`

File: src/bundle/Resources/translations/ezrepoforms_role.en.xliff [hunk 3]
```diff
@@ -37,18 +37,18 @@
...
-        <source>role.policy.limitation.location.udw_title</source>
-        <target state="new">role.policy.limitation.location.udw_title</target>
+        <source>Select locations to use as policy limitation</source>
+        <target state="new">Select locations to use as policy limitation</target>
         <note>key: role.policy.limitation.location.udw_title</note>
       </trans-unit>
       <trans-unit id="be8159ab7985797bd281cb0f7f58bb4d2c925159" resname="role.policy.limitation.not_implemented">
-        <source>role.policy.limitation.not_implemented</source>
-        <target state="new">role.policy.limitation.not_implemented</target>
+        <source>Limitation edition for '%limitationTypeIdentifier%' is not implemented yet.</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/ezrepoforms_role.en.xliff]: `Editing limitations for ...`

**Comment by mikadamczyk** [comment]
Fixed and ready to tests

**Pull request merged**